spec/dummy_nanoc/README.md

```diff
@@ -30,11 +30,11 @@ Live demo link :
 
 Note : There is some problem in nyaplot (in live link. It works fine locally). Some css is not working so some styling doesn't working properly. You can see it properly in local setup.
 
-[index.html](https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/)
+[index.html](https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/)
 
 
-[nyaplot](https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/nyaplot)
+[nyaplot](https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/nyaplot)
```

Comment:
Please change it in spec/dummy_nanoc example as well : https://github.com/SciRuby/daru-view/blob/master/spec/dummy_nanoc/content/index.html#L21

and generate a updated output files for nanoc app using the command `bundle exec nanoc view`